index.html

```diff
@@ -19,23 +19,170 @@
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
         </ul>
-    </section>
+      </div>
+      <button type="button" class="btn">See Project</button>
+    </article>
+
+  <!-- CARDS -->
+<section class="grid-container">
+  <section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section>
+
+  <section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
   </section>
+</section>
+</section>
+<section class="about-me">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![zzz](https://user-images.githubusercontent.com/69638013/160237315-65772a50-f9a7-47f4-924b-718bd6b9efbb.png)  |  ![Screenshot_14](https://user-images.githubusercontent.com/69638013/160237311-7c62d27d-1d0a-409c-9492-0ed60e4ffe13.png)


- [ ] Please add the appropriate padding before the header tag
![vvv](https://user-images.githubusercontent.com/69638013/160237384-8d6a9ea2-1a96-417f-a791-39e8477813f5.png)

- [ ] Please ensure to you use the right colors stated in the Figma design for when the button is clicked or hovered, please do this for all buttons in your design, check screenshot above ⬆️